instructor/patch.py

```diff
@@ -26,6 +27,7 @@
     ChatCompletionMessageParam,
 )
 from openai.types.completion_usage import CompletionUsage
+from anthropic.types.message import Message
```

Comment:
avoid the import  via importlib

and we can make a group for anthropic in the toml